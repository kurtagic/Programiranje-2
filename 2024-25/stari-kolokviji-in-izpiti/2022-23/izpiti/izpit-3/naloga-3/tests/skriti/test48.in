vtjDfMqfINfO0c8St9GgnuXI3m6LzdcTx3D5kmHkyYM9PmuLLCvMKLUDdQlFzK1ejDyx4GtRjKBeEEAYJsb86rlROknPrf7VfqjcFzMcqviTwOmYACVtV5VM3W9zNNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
