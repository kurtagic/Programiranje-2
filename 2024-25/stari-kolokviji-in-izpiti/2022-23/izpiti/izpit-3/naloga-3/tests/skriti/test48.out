vtjDfMqfINfO0c8St9GgnuXI3m6LzdcTx3D5kmHkyYM9PmuLLCvMKLUDdQlFzK1ejDyx4GtRjKBeEEAYJsb86rlROknPrf7VfqjcFzMcqviTwOmYACVtV5VM3W9zNNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
vtjDfMqfINfO0c8St9GgnuXI3m6L
v
 tjDfMqfINfO0c8St9GgnuXI3m6L
 tjDfMqfINfO0c8St9Ggn
 t
  jDfMqfINfO0c8St9Ggn
  jDfMqfINfO0c8S
  jDfM
  j
   DfM
   D
    f
     M
      q
       fINfO0c8S
       f
        INfO0c8S
        IN
        I
         N
          f
           O0c8S
           O0
           O
            0
             c
              8S
              8
               S
                t
                 9Ggn
                 9Gg
                 9G
                 9
                  G
                   g
                    n
                     u
                      XI3m6L
                      XI3
                      X
                       I3
                       I
                        3
                         m
                          6L
                          6
                           L
                            z
                             dcTx3D5kmHkyYM9PmuLLCvMKLUDdQlFzK1ejDyx4GtRjKBeEEAYJsb86rlROknPrf7VfqjcFzMcqviTwOmYACVtV5VM3W9zNNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                             dcTx3D5kmHkyYM9PmuLLCvMKLUDdQlF
                             dcTx3D5kmHk
                             dcT
                             d
                              cT
                              c
                               T
                                x
                                 3D5kmHk
                                 3D5k
                                 3D5
                                 3
                                  D
                                   5
                                    k
                                     m
                                      Hk
                                      H
                                       k
                                        y
                                         YM9PmuLLCvMKLUDdQlF
                                         YM9PmuLLC
                                         YM9Pm
                                         YM9P
                                         Y
                                          M9P
                                          M9
                                          M
                                           9
                                            P
                                             m
                                              u
                                               LLC
                                               L
                                                LC
                                                L
                                                 C
                                                  v
                                                   MKLUDdQlF
                                                   MKLUDdQ
                                                   MKLUD
                                                   MKL
                                                   M
                                                    KL
                                                    K
                                                     L
                                                      U
                                                       D
                                                        d
                                                         Q
                                                          l
                                                           F
                                                            z
                                                             K1ejDyx4GtRjKBeEEAYJsb86rlROknPrf7VfqjcFzMcqviTwOmYACVtV5VM3W9zNNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                                                             K1ejDyx4GtRjKBeEEAYJsb86rlROknPrf7VfqjcF
                                                             K1ejD
                                                             K1e
                                                             K1
                                                             K
                                                              1
                                                               e
                                                                j
                                                                 D
                                                                  y
                                                                   x4GtRjKBeEEAYJsb86rlROknPrf7VfqjcF
                                                                   x
                                                                    4GtRjKBeEEAYJsb86rlROknPrf7VfqjcF
                                                                    4G
                                                                    4
                                                                     G
                                                                      t
                                                                       RjKBeEEAYJsb86rlROknPrf7VfqjcF
                                                                       RjKBeEEAYJ
                                                                       R
                                                                        j
                                                                         KBeEEAYJ
                                                                         KB
                                                                         K
                                                                          B
                                                                           e
                                                                            EEAYJ
                                                                            EEA
                                                                            E
                                                                             EA
                                                                             E
                                                                              A
                                                                               Y
                                                                                J
                                                                                 s
                                                                                  b86rlROknPrf7VfqjcF
                                                                                  b86
                                                                                  b
                                                                                   86
                                                                                   8
                                                                                    6
                                                                                     r
                                                                                      lROknPrf7VfqjcF
                                                                                      lROknP
                                                                                      lROk
                                                                                      l
                                                                                       ROk
                                                                                       RO
                                                                                       R
                                                                                        O
                                                                                         k
                                                                                          n
                                                                                           P
                                                                                            r
                                                                                             f7VfqjcF
                                                                                             f7Vf
                                                                                             f
                                                                                              7Vf
                                                                                              7V
                                                                                              7
                                                                                               V
                                                                                                f
                                                                                                 q
                                                                                                  jcF
                                                                                                  j
                                                                                                   cF
                                                                                                   c
                                                                                                    F
                                                                                                     z
                                                                                                      McqviTwOmYACVtV5VM3W9zNNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                                                                                                      McqviTwOmYACVtV5VM3W9
                                                                                                      McqviT
                                                                                                      Mcq
                                                                                                      Mc
                                                                                                      M
                                                                                                       c
                                                                                                        q
                                                                                                         v
                                                                                                          iT
                                                                                                          i
                                                                                                           T
                                                                                                            w
                                                                                                             OmYACVtV5VM3W9
                                                                                                             OmYACV
                                                                                                             O
                                                                                                              m
                                                                                                               YACV
                                                                                                               Y
                                                                                                                ACV
                                                                                                                AC
                                                                                                                A
                                                                                                                 C
                                                                                                                  V
                                                                                                                   t
                                                                                                                    V5VM3W9
                                                                                                                    V5VM3
                                                                                                                    V
                                                                                                                     5VM3
                                                                                                                     5
                                                                                                                      V
                                                                                                                       M3
                                                                                                                       M
                                                                                                                        3
                                                                                                                         W
                                                                                                                          9
                                                                                                                           z
                                                                                                                            NNH1GVuzVYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                                                                                                                            NNH1GVu
                                                                                                                            NNH1GV
                                                                                                                            NNH1G
                                                                                                                            N
                                                                                                                             NH1G
                                                                                                                             N
                                                                                                                              H1G
                                                                                                                              H
                                                                                                                               1G
                                                                                                                               1
                                                                                                                                G
                                                                                                                                 V
                                                                                                                                  u
                                                                                                                                   z
                                                                                                                                    VYrfFc3NQFfyktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                                                                                                                                    VYrfFc3NQFf
                                                                                                                                    VY
                                                                                                                                    V
                                                                                                                                     Y
                                                                                                                                      r
                                                                                                                                       fFc3NQFf
                                                                                                                                       f
                                                                                                                                        Fc3NQFf
                                                                                                                                        Fc3NQF
                                                                                                                                        F
                                                                                                                                         c
                                                                                                                                          3NQF
                                                                                                                                          3N
                                                                                                                                          3
                                                                                                                                           N
                                                                                                                                            Q
                                                                                                                                             F
                                                                                                                                              f
                                                                                                                                               y
                                                                                                                                                ktLjT9Y3MfkYL0tq55NpDa2UnlmN5xZwAXhD
                                                                                                                                                ktLjT9Y3MfkYL0tq55NpDa2UnlmN5
                                                                                                                                                k
                                                                                                                                                 t
                                                                                                                                                  LjT9Y3MfkYL0tq55NpDa2UnlmN5
                                                                                                                                                  LjT9Y3MfkYL0
                                                                                                                                                  LjT9Y3Mf
                                                                                                                                                  L
                                                                                                                                                   j
                                                                                                                                                    T9Y3Mf
                                                                                                                                                    T9Y3M
                                                                                                                                                    T9
                                                                                                                                                    T
                                                                                                                                                     9
                                                                                                                                                      Y
                                                                                                                                                       3M
                                                                                                                                                       3
                                                                                                                                                        M
                                                                                                                                                         f
                                                                                                                                                          k
                                                                                                                                                           YL0
                                                                                                                                                           Y
                                                                                                                                                            L0
                                                                                                                                                            L
                                                                                                                                                             0
                                                                                                                                                              t
                                                                                                                                                               q55NpDa2UnlmN5
                                                                                                                                                               q
                                                                                                                                                                55NpDa2UnlmN5
                                                                                                                                                                55N
                                                                                                                                                                55
                                                                                                                                                                5
                                                                                                                                                                 5
                                                                                                                                                                  N
                                                                                                                                                                   p
                                                                                                                                                                    Da2UnlmN5
                                                                                                                                                                    Da2U
                                                                                                                                                                    D
                                                                                                                                                                     a
                                                                                                                                                                      2U
                                                                                                                                                                      2
                                                                                                                                                                       U
                                                                                                                                                                        n
                                                                                                                                                                         lmN5
                                                                                                                                                                         l
                                                                                                                                                                          m
                                                                                                                                                                           N5
                                                                                                                                                                           N
                                                                                                                                                                            5
                                                                                                                                                                             x
                                                                                                                                                                              ZwAXhD
                                                                                                                                                                              Z
                                                                                                                                                                               w
                                                                                                                                                                                AXhD
                                                                                                                                                                                AX
                                                                                                                                                                                A
                                                                                                                                                                                 X
                                                                                                                                                                                  h
                                                                                                                                                                                   D
